src/main/webapp/i18n/de/dataExport.json

```
@@ -2,14 +2,24 @@
     "artemisApp": {
         "dataExport": {
             "title": "Datenexport",
+            "titleDownload": "Datenexport herunterladen",
             "description": "Hier kannst du einen Datenexport gemäß Art.15 DSGVO Auskunftsrecht der betroffenen Person anfordern. Sobald der Export erstellt wurde, erhältst du eine Email mit einem Link um diesen herunterzuladen. Du kannst den Export sieben Tage lang herunterladen, danach wird er gelöscht. ",
+            "descriptionDownload": "Hier kannst du deinen angeforderten Datenexport herunterladen.",
             "request": "Datenexport anfordern",
             "download": "Datenexport herunterladen",
             "requestSuccess": "Datenexport erfolgreich angefordert",
             "downloadSuccess": "Datenexport erfolgreich heruntergeladen",
             "typeLoginToConfirm": "Bitte gib deine Login-Kennung ein, um den Datenexport zu bestätigen",
             "confirmationQuestion": "Willst du wirklich den Datenexport anfordern?",
-            "confirmationHeader": "Datenexport anfordern"
+            "confirmationHeader": "Datenexport anfordern",
+            "requestError": "Datenexport konnte nicht angefordert werden. Bitte versuche es später erneut.",
+            "downloadError": "Datenexport konnte nicht heruntergeladen werden. Bitte versuche es später erneut.",
+            "recentlyRequested": "Du hast kürzlich einen Datenexport angefordert. Deshalb kannst du aktuell keinen Datenexport anfordern.",
+            "notDownloadable": "Dieser Datenexport kann nicht mehr heruntergeladen werden, da er nicht (mehr) existiert.",
+            "mostRecentNotDownloadable": "Dein aktuellster Datenexport kann nicht heruntergeladen werden, da er entweder bereits gelöscht wurde oder noch nicht erstellt wurde.",
```

There should always only be one active report, so this distinction is unnecessary imo.
```suggestion
            "mostRecentNotDownloadable": "Dein Datenexport kann nicht heruntergeladen werden, da er entweder bereits gelöscht wurde oder noch nicht erstellt wurde.",
```